recipes-kernel/linux/linux-milkv-duo-dev.bb

```diff
@@ -0,0 +1,47 @@
+SUMMARY = "Milk-V Duo dev kernel recipe"
+
+inherit kernel
+require recipes-kernel/linux/linux-yocto.inc
+
+DEPENDS = "u-boot-mkimage-native dtc-native"
+
+BRANCH ?= "main"
+FORK ?= "xyq1113723547"
```

Comment:
this seems your local fork, what the plan to upstream it ?

Reply:
Personally I have no plan to upstream my fork. Sophgo is hiring for people who can upstream their kernel. Another team is also trying to do kernel upstream here: https://github.com/duosdk/linux/tree/duosdk. I provide my fork here because the kernel in the official SDK can not be built by current upstream toolchain because of Kconfig and Makefiles. Also, dtb generation is out of tree, so is not suitable in yocto's workflow right now.

Reply:
OK as long as your fork is kept alive we can think about using it